test/sql-luatest/gh_6766_mp_ext_via_netbox_test.lua

```diff
@@ -0,0 +1,49 @@
+local server = require('test.luatest_helpers.server')
+local net = require('net.box')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'gh-6766'})
+    g.server:start()
+end)
+
+g.after_all(function()
+    g.server:stop()
+end)
+
+g.test_6766_1 = function()
+    local conn = net.connect(g.server.net_box_uri)
```

Comment:
You can also use `g.server.net_box` - it is a pre-created connection. Up to you.

Reply:
Thank you! I used `g.server.net_box` instead of `net.connect()` and dropped `conn:close()`.